### PR TITLE
[TACHYON-1476] Cleanup of Tachyon server test code

### PR DESCRIPTION
**Pull request by jsimsa**
Addresses: https://tachyon.atlassian.net/browse/TACHYON-1476

### CODE REVIEW

**Comment by aaudiber** [comment]
LGTM, thanks for the cleanup. I've left some minor comments

File: servers/src/main/java/tachyon/master/MasterSource.java [hunk 1]
```diff
@@ -25,6 +25,8 @@
 import com.codahale.metrics.MetricRegistry;
 
 import tachyon.Constants;
+import tachyon.master.MasterContext;
```
Comment (yupeng9) [inline servers/src/main/java/tachyon/master/MasterSource.java]: Why are these added?

File: servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java [hunk 2]
```diff
@@ -1,87 +0,0 @@
-/*
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java]: Why is this deleted?

File: servers/src/main/java/tachyon/master/MasterSource.java (hunk 1 above)
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/MasterSource.java]: Good catch. I think IntelliJ goofed up here.

File: servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java [hunk 3]
```diff
@@ -20,8 +20,6 @@
 
 import tachyon.Constants;
 import tachyon.master.block.BlockId;
-import tachyon.master.file.meta.InodeDirectory;
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java]: hmm, don't we have a rule for checking unnecessary imports?

File: servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java [hunk 4]
```diff
@@ -28,7 +28,7 @@
 import tachyon.exception.BlockInfoException;
 
 /**
- * Unit tests for {@link tachyon.master.file.meta.InodeFile}
+ * Unit tests for {@link InodeFile}
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java]: Add a period

File: servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java (hunk 2 above)
Comment (jsimsa) [inline servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java]: because there is an identical `MasterBlockInfoTest.java` file already ... unless I am missing something

**Comment by yupeng9** [comment]
Thanks for the cleanup. I left some comments.

File: servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java (hunk 4 above)
Comment (jsimsa) [inline servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java]: Done.

File: servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java (hunk 3 above)
Comment (jsimsa) [inline servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java]: yes, I am not sure why it didn't catch this

Comment (jsimsa) [inline servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java]: ah, technically this import is used ... but it is not needed because the test class is in the same package

**Comment by aaudiber** [comment]
+1 LGTM

File: servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java (hunk 2 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java]: gotcha.

File: servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java (hunk 3 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java]: I see

**Comment by yupeng9** [comment]
LGTM
